presidio_evaluator/data_generator/faker_extensions/span_generator.py

```diff
@@ -31,14 +31,19 @@ class SpanGenerator(Generator):
     """
```

Comment:
Changes here: when generating data, also include the sample_id to be able to later track back to it.